wallet/wallet.c

```diff
@@ -16,6 +16,17 @@
 #define DIRECTION_INCOMING 0
 #define DIRECTION_OUTGOING 1
 
+static void outpointfilters_init(struct wallet *w)
+{
+	struct utxo **utxos = wallet_get_utxos(NULL, w, output_state_any);
```

Comment:
We will monitor all outputs, even those already spent?

Reply:
I'm planning to have a cleanup that triggers periodically and garbage collects spent outputs every once in a while if they are deeply buried, i.e., cannot be reorged out. That'll drop from the DB and from the filter set.

Reply:
okay, seems good.

Reply:
Yep, we treat 100 as "forever" elsewhere, so that seems a good benchmark.